bootcompiler/src/main/scala/org/mozartoz/bootcompiler/Main.scala

```diff
@@ -211,7 +220,7 @@ object Main {
    *  @param moduleDefs list of files that define builtin modules
    */
   private def loadModuleDefs(prog: Program, moduleDefs: List[String]) = {
-    JSON.globalNumberParser = (_.toInt)
+    // JSON.globalNumberParser = (_.toInt)
 
```

Comment:
I guess this is not needed anymore because you're using spray-json now? In that case, it should be removed, not just commented out.